Userland/Applications/Presenter/SlideObject.cpp

```diff
@@ -49,6 +49,30 @@ ErrorOr<NonnullRefPtr<SlideObject>> SlideObject::parse_slide_object(JsonObject c
 SlideObject::SlideObject()
 {
     REGISTER_RECT_PROPERTY("rect", rect, set_rect);
+
+    register_property(
+        "frames",
+        [this] {
+            auto const& frames = this->frames();
+            JsonArray json_frames;
+            for (auto element : frames)
+                json_frames.append(element);
+
+            return json_frames;
+        },
+        [this](auto& value) {
+            HashTable<unsigned> frames;
+            if (value.is_array()) {
+                auto const& values = value.as_array().values();
+                for (auto const& element : values)
+                    frames.set(element.template to_number<unsigned>(0));
+            } else {
+                return false;
+            }
```

Comment:
Early return feels a bit more Serenity-ish here, and reduces rightward drift:
```suggestion
            if (!value.is_array())
                return false;

            auto const& values = value.as_array().values();
            for (auto const& element : values)
                frames.set(element.template to_number<unsigned>(0));
```